service/frontend/workflow_handler.go

```diff
@@ -4763,6 +4598,23 @@ func (wh *WorkflowHandler) StartBatchOperation(
 	return &workflowservice.StartBatchOperationResponse{}, nil
 }
 
+func snakeCaseBatchType(batchType enumspb.BatchOperationType) string {
```

Comment:
To maintain consistency with how we do these now a few of the batch operation types needed special casing